format-definitions/parsec-platform-evidence-tpm.md

```diff
@@ -0,0 +1,94 @@
+# Parsec TPM explicit platform attestation
+
+This document covers the technical details of explicit platform attestation
+using a TPM2.0.
+
+The attestation and verification process revolve around a signed attestation
+token, and the extra data required to verify it.
+
+## TPM Platform Attestation Statement Format
+
+While for key attestation we can use the format supplied by WebAuthn for that
+use-case, and while other WebAuthn attestation statement formats are defined for
+platform attestations such as the [Android SafetyNet
+format](https://www.w3.org/TR/webauthn-2/#sctn-android-safetynet-attestation),
+no such format exists for TPM platform attestation. However, since the format
+was made with extensibility in mind, defining our own format within the overall
+scheme is straight-forward.
+
+```
+parsecTpmPlatStmtFormat = {
+                 tpmVer: "2.0",
+                 alg: COSEAlgorithmIdentifier,
```

Comment:
As discussed with @ionut-arm we do not need to encode the alg, separately in the "platform" statement as the algorithm is already provided in the sig bytes encoded by TPM